### ISSUE TITLE
Quest v0.1 / G6PD: Patient Profile View

### ISSUE THREAD

**Issue by f-odhiambo** [opensrp/fhircore#585]
**Patient Profile View:** when selecting a patient from the list view, the user can see that patient's profile.  
For the moment, the patient profile will be quite simple - we just need to visualize the patient's demographic information. For this scope, we don't need to implement editing or removing patients unless we get this for free. 

(mock from G6DP, exact details TBC)
<img width="223" alt="Screen Shot 2021-09-22 at 1 22 23 PM" src="https://user-images.githubusercontent.com/12836913/134416265-8a6ca3d2-bd7f-463b-95b4-ba87ef8b7cc6.png">

**Comment by f-odhiambo** [comment]
This profile view has been updated when being viewed for G6PD 

**See mock**
<img width="425" alt="Screen Shot 2021-10-05 at 11 32 42 PM" src="https://user-images.githubusercontent.com/4540684/136098538-1d708f69-b434-491d-98c8-86357a33494c.png">

**Comment by pld** [comment]
I think this issue is 2 issues

1) the first generic version that's closer to the screenshots in the issue description.

2) a more use-case specific version, like in the above screenshot that needs a configuration to control what's displayed and is not yet scoped

@syedowaisali90 please only concern yourself with the generic version for now, @f-odhiambo can you create a new issue for the configured version that's marked blocked and needs scoping?

But, for the generic version, what do we display? Is it like the final screen but it shows all the forms, doesn't show anything under the patients name, and only has the date for previous result?

**Comment by pld** [comment]
I may also be missing information here, are we far enough along with configuration that we do these as a single issue? If so that's great, and if so @syedowaisali90 can you add more details on the implementation plan for doing this through configuration so we can discuss and give feedback before work begins? Thanks

**Comment by syedowaisali90** [comment]
@pld I don't think it will be controlled by configs and I think the updated mock that @f-odhiambo has shared looks like a more updated version and if not so then we need more context.

**Comment by pld** [comment]
OK, more context is needed then. The updated mock is specific to g6pd, which this is not supposed to be, so we'd want to implement a version of this that is not-g6pd specific and if we need anything g6pd specific that would have to be controlled through dynamic files loaded via sync.

@rowo @f-odhiambo can we build a generic version of this that is effective for g6pd? If not, let's schedule a meeting to define the implementation plan for how to have the g6pd portions of this loaded dynamically so this stays in-line w/the single APK approach.

**Comment by isabelcshaw** [comment]
@owais-vd after discussion with @pld we've aligned on this version of the profile view that only has one addition for G6PD. Some notes on the image below: 
1. I've removed the G6PD status on the client's name (sorry it isn't neater) - you can consider that out of scope

<img width="220" alt="Screen Shot 2021-10-07 at 11 24 07 AM" src="https://user-images.githubusercontent.com/12836913/136441513-2865b995-7945-4a2c-b97b-81592351a34e.png">

**Comment by isabelcshaw** [comment]
@pld updated the comment and image - let me know if anything is missing

**Comment by owais-vd** [comment]
Noted. Thanks, @pld @isabelcshaw

**Comment by f-odhiambo** [comment]
**Update**
1. Updated the UI as per new mock and load test forms from configs and dynamically populated the test results.
2.  Adding test cases and then create a PR for review.

**Comment by pld** [comment]
What resource are you storing based on what's submitted to the Questionnaire?

**Comment by owais-vd** [comment]
ok cool, then i need to update the PR. Thanks
